### ISSUE TITLE
Feature: Rowing competition

### ISSUE THREAD

**Issue by svenmeier** [svenmeier/coxswain#27]
With #13 solved, we could easily allow competing in groups.

Multiple devices could automatically be linked through UDP packages, showing the delta (time and/or distance) to the leading rower.

We would need two Waterrowers to test this though.

**Comment by frankHarmann** [comment]
Sounds interesting. But maybe https://developers.google.com/games/services/android/realtimeMultiplayer would be a more flexible solution as the chances your friends will bring their water rower along are rather low compared to an online match. The Google play app looks quiet interesting for this as there is no need to host a central server.

**Comment by svenmeier** [comment]
You're right, Google Play's multiplayer would be a great solution to support remote competitions.

**Comment by gogolo2** [comment]
Would be a nice feature, 2 considerations: 

- Waterlevel has to be calculated (e.g. I have 17 Litre) 
- I'm on CM with no gapps, Would be nice to minimize google dependencies.

**Comment by mdond** [comment]
I don't know if this is possible at all by an android app, but a connection to Zwift would be a nice thing.

Here one is described by using a Raspberry Pi:
http://www.olympum.com/sports/rowing-with-waterrower-in-zwift/

It should be possible by implementing the protocoll between Zwift Mobile Link and Zwift in Coxswain, but I don't know if the protocol is documented.

**Comment by svenmeier** [comment]
To my knowledge Zwift doesn't offer an API that Coxswain could utilize :(.

**Comment by kruskaal** [comment]
Hi Sven, Thanks very much for your App! 
It would indeed be fun to compete with other users like on the web App We-Row . Did you look into the google play API suggested by FrankHarmann?
I am not an expert in Android Apps development but if I can give a hand let me know.

**Comment by svenmeier** [comment]
I didn't invest any time in this yet, although Google Play API is probably the easiest solution.
But personally I didn't use any App yet, that does something similar. So I'm not sure how players will be able to connect and team up.

**Comment by f-bader** [comment]
Is this still under consideration?

**Comment by svenmeier** [comment]
Hi Fabian,

yes, this is definitely on my wish-list.

No ETA currently though.
Sven
